### PR TITLE
How We Use Github

### PR DESCRIPTION
**Pull request by Snailed**

### CODE REVIEW

**Review by nikolajvonholck — APPROVED** [review]
Great! I have pushed some minor changes. I you agree with them, then merge – otherwise, let's discuss.

File: how-we-use-github.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# How we use GitHub (and Git in general)
+We use Git for version controlling our code and GitHub for hosting our repositories, running CI/CD tasks and managing pull requests, issues and code reviews. All code must be stored on GitHub. [Here is a guide for learning how to use git](https://rogerdudler.github.io/git-guide/).
+
+## Teams
```
Comment (widding) [inline how-we-use-github.md:4]: Suggestion:

How about we use teams for projects instead? That way, project owners wont need to add a new 'team member' to each repository associated with the project. Eg. Mikkeller retainer requires access to 2-3 repositories.

File: how-we-use-github.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+If you work 15 hours or more each week as a developer at Kvalifik then you should be an owner of our GitHub organization. Otherwise, you will get access to the individual repositories as needed – some of these repositories are grouped in teams, for instance "Mikkeller". All interns should be members of the 'Interns' team granting them access to a limited subset of repositories.
+
+## Branches
+We use Gitflow as our branching strategy. Read more about it [here](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
+Each repository has a default branch. This should be set to a branch named `staging`, and should represent the state of the staging environment of the application (called `develop` in the Gitflow documentation).
+Each repository should also have a branch called `production`, which stores our production code (called `main` in the Gitflow documentation).
+Each feature branch should have the name `feature/[issue-id]-[title]` corresponding to the Github issue ID and the title of the issue. *You may only push to a feature branch, never directly to staging nor production*.
```
Comment (widding) [inline how-we-use-github.md:11]: Did we agree on a naming convention for the different types of branches beyond 
'feature'?

Comment (Snailed) [reply how-we-use-github.md:11]: Hmm, I usually use `fix/[branch-name]` for UX fixes that are not its own user story. I believe that you are thinking about the `hotfix/` prefix as described [here](https://zellwk.com/blog/git-flow/#hotfix-branches), but these are meant for fixes that skip `staging` goes straight to `production`. I don't believe we should encourage such practice, especially when we have an efficient CI/CD flow. What do you think?

File: how-we-use-github.md (hunk 1 above)
Comment (Snailed) [reply how-we-use-github.md:4]: I agree, will add when I have the time

**Comment by Snailed** [comment]
Add comment about how we do not squash our commits, but that we used to :)

File: how-we-use-github.md (hunk 2 above)
Comment (nikolajvonholck) [reply how-we-use-github.md:11]: I don't think we need more than `feature/[branch-name]` for now. It makes these branches clearly different from the branches `production` and `staging` which is really what we use them for right now.

**Comment by widding** [comment]
@Snailed I tried re-writing the teams section per our discussion earlier, please give it a look.

**Comment by Snailed** [comment]
@widding looks good, now we just need the squash comment :+1:

**Comment by widding** [comment]
> @widding looks good, now we just need the squash comment 👍

Do you want me to add it or are you on it?

**Pull request merged**